source/agora/node/BlockStorage.d

```diff
@@ -784,13 +784,22 @@ public class BlockStorage : IBlockStorage
 
             size_t record_size = (size_t.sizeof * 2 + Hash.sizeof);
             size_t record_count = idx_file.size / record_size;
+
```

Comment:
Could you reword the commit message a bit?

> Replace readSizeType() method and read the index file directly

Reply:
Fixed.